samples/04_gis_analysts_data_scientists/model_explainability_using_shap_for_tabular_data.ipynb

```diff
@@ -0,0 +1,534 @@
+{
```

Comment:
<p>Similar to feature importance in sklearn) - &gt; no need to cap S in Similar. Can you add a hyperlink to the said topic in sklearn?</p><p>This plot tells the -&gt; The plot below (for clarity)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Done</p>